apps/docs/pages/guides/integrations/snaplet.mdx

```diff
@@ -0,0 +1,151 @@
+
+- Subsetting (skipping will create a snapshot with all your data).
+- Transforming (skipping will leave all your table columns untouched).
+
+Once you have gone through those steps, a new snapshot process will start and you will now be on the “capture step.” Wait for it to finish - this is the snapshot you will restore into your **target database.**
+
+### Step 3: Create a target database on Supabase (Development Database) [#create-target-db]
+
+Your target database is where you want Snaplet to restore the captured snapshot of your production project. This would most likely be either your staging or development Supabase project.
+
+If you don’t already have a development database setup on Supabase, you’ll need to create a new database by setting up a new project on Supabase. To create a new project you’ll have to do the following:
+
+1. Go to [app.supabase.io](https://app.supabase.io/)
+
+2. Go to your **organization > All projects**
+
+3. In the top left click on your current project name and new project:
+  ![New project](/docs/img/guides/integrations/snaplet/new_project.webp)
+
+4. Wait for the new database to launch
+
+> Remember the password you use when creating the project. You’ll need this password to connect your database to Snaplet later.
+
+### Step 4: Install the Snaplet CLI [#cli-installation]
+
+1. Open your terminal and run `npm install -g snaplet`
+2. Run `snaplet auth login`
+3. Navigate to `https://app.snaplet.dev/access-token/cli` to get your access token.
+4. Paste the access token in the terminal.
+
+### Step 5: Config setup [#config-setup]
+
+You're now ready to restore your production snapshot into your Supabase development project.
+
+1. Navigate to your project directory
+
+2. Run `snaplet config setup`
+   i. You will be shown a warning to write create `.snaplet/config` in your project directory. Select yes (y) to create the file.
+   ii. Then you will be asked for a **target database connection string.** These are the database credentials of your **target database that we set up in step 3** (your staging or development database).
+
+   > Note that you need to be using the `postgres` user for your development environment
+
+3. Run `snaplet project setup` - you will be presented with a list of projects, these are databases that are connected to your Snaplet account. Choose the project that contains the snapshot you created in Step 2.
+
+### Step 6: Restore the data target [#restore-data-target]
+
+With all the above steps complete, we can now restore!
+
+1. Run `snaplet snapshot restore --no-reset`
+
+> **Note on `--no-reset`:** This will skip the “reset” step of the restore command. So no existing schemas will be dropped. Learn more about data operations [here](https://docs.snaplet.dev/getting-started/restoring#opting-out)
+
+## All done!
+
+If you want to learn more about Snaplet, you can explore our docs. If you have any questions, feel free to reach out on [Discord](https://app.snaplet.dev/chat).
```

Comment:
This sounds like it was copied over from the Snaplet docs. Can you make it more third-party? 'Our' docs here would refer to Supabase, which I'm guessing is not what you want :D